README.md

```diff
@@ -101,12 +101,16 @@ On Red Hat-based operating systems, including Fedora, install gettext via
 
 1. In your `project.clj`, add `puppetlabs/i18n` to the `:dependencies` and to
    the `plugins`
-1. Run `lein i18n init`. This will
```

Comment:
True, on the other hand having the real indices in there doesn't break anything and makes the README.md source IMO a little bit more readable.